priv/perf/apps/load_test/lib/scenario/account_transactions.ex

```diff
@@ -0,0 +1,271 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.Scenario.AccountTransactions do
+  @moduledoc """
+  This scenario tests watcher info apis when number of transaction increases for an account.
+  """
+
+  use Chaperon.Scenario
+
+  alias Chaperon.Timing
+  alias LoadTest.Connection.WatcherInfo, as: Connection
+  alias LoadTest.Service.Faucet
+  alias LoadTest.Ethereum.Account
+  alias LoadTest.Utils.Encoding
+  alias WatcherInfoAPI.Api
+  alias WatcherInfoAPI.Model
+
+  @poll_interval 15_000
+  @default_retry_attempts 15
+
+  @eth <<0::160>>
+  @test_output_amount 1
+
+  @spec init(Chaperon.Session.t()) :: Chaperon.Session.t()
+  def init(session) do
+    session
+    |> log_info("start init with random delay...")
+    |> random_delay(Timing.seconds(5))
+  end
+
+  def run(session) do
+    iterations = config(session, [:iterations])
+    fee_wei = Application.fetch_env!(:load_test, :fee_wei)
+
+    amount = iterations * (@test_output_amount + fee_wei)
+    {:ok, sender} = Account.new()
+    {:ok, _} = Faucet.fund_child_chain_account(sender, amount, @eth)
+
+    {:ok, faucet} = Faucet.get_faucet()
+
+    session
+    |> assign(faucet: faucet, iteration: 1)
+    |> wait_for_balance_update(sender)
+    |> log_info("user created: " <> Encoding.to_hex(sender.addr))
+    |> repeat(:repeat_task, [sender], iterations)
+    |> log_info("end...")
+  end
+
+  def wait_for_balance_update(session, sender, retry \\ @default_retry_attempts) do
+    {:ok, session} = do_wait_for_balance_update(session, sender, retry)
+    session
+  end
+
+  def repeat_task(session, sender) do
+    session
+    |> log_info("running iteration: " <> Integer.to_string(session.assigned.iteration))
+    |> retry_on_error(:test_apis, [sender], retries: @default_retry_attempts, random_delay: seconds(30))
+    |> update_assign(iteration: &(&1 + 1))
+  end
+
+  def test_apis(session, sender) do
+    session
+    |> test_api_account_get_balance(sender)
+    |> test_api_account_get_utxos(sender)
+    |> test_api_account_get_transactions(sender)
+    |> test_api_account_create_and_submit_transactions(sender)
+  end
+
+  defp test_api_account_get_balance(session, sender) do
+    start = Timing.timestamp()
+    {:ok, _} = get_balance(sender)
+
+    add_metric(
+      session,
+      {:call, {LoadTest.Scenario.AccountTransactions, '/account.get_balance'}},
+      Timing.timestamp() - start
+    )
+  end
+
+  defp test_api_account_get_utxos(session, sender) do
```

Comment:
all the`test_api_account` functions are very similar. How about introducing a function:
```
defp measure(session, sender, api_call, metric_name) do
    start = Timing.timestamp()
    {:ok, _} = api_call.(sender)

    add_metric(
      session,
      {:call, {LoadTest.Scenario.AccountTransactions, metric_name}},
      Timing.timestamp() - start
    )
  end
```
and then `test_api_*` would be like:
```
defp test_api_account_get_balance(session, sender) do
  measure(session, sender,  &get_balance/1, `/account.get_balance`)
end
```